### PR TITLE
finished challenge

### PR DESCRIPTION
**Pull request by Nhuuu**

### CODE REVIEW

**Comment by Nhuuu** [comment]
- [ x] Top-level README “Table of Contents” is updated
 - [x ] Feature tasks for this challenge are completed
 - [ x] Unit tests written and passing
     - [x ] “Happy Path” - Expected outcome
     - [ x] Expected failure
     - [ x] Edge Case (if applicable/obvious)
 - [x ] README for this challenge is complete
     - [ x] Summary, Description, Approach & Efficiency, Solution
     - [ x] Link to code
     - [ x] Picture of whiteboard

File: Data-Structures/src/main/java/utilities/MultiBracketValidation.java [hunk 1]
```diff
@@ -1,29 +1,22 @@
...
+public class MultiBracketValidation {
 
-        }
+  public static boolean validate(String input){
+    Stack<Character> openBrackets = new Stack<>();
+    for(int i = 0; i < input.length(); i++){
+      char current = input.charAt(i);
+      if(current == '(' || current == '[' || current == '{'){
+        openBrackets.push(current);
+      } else if (current == ')' && openBrackets.peek() == '(' || current == ']' && openBrackets.peek() == '[' || current == '}' && openBrackets.peek() == '{'){
```
Comment (evanslaton) [inline Data-Structures/src/main/java/utilities/MultiBracketValidation.java:13]: For these really long conditionals you can put each on it's own line to make it a little easier to read. This is totally optional so if you hate it ignore this!

```
else if (current == ')' && openBrackets.peek() == '(' 
     || current == ']' && openBrackets.peek() == '[' 
     || current == '}' && openBrackets.peek() == '{') {
```
